index.bs

```diff
@@ -551,25 +597,32 @@ spec:css-syntax-3;
 
   <div algorithm="relevant credential interfaces">
     The <dfn for="CredentialRequestOptions">relevant credential interface objects</dfn> for a given
-    {{CredentialRequestOptions}} (|options|) is a set of [=interface objects=], collected as follows:
+    {{CredentialRequestOptions}} or {{CredentialRequestOptions}} (|options|) is a set of [=interface objects=], collected as follows:
```

Comment:
Was this change intended? It looks like you meant to write something else?

```suggestion
    {{CredentialRequestOptions}} (|options|) is a set of [=interface objects=], collected as follows:
```

Reply:
ah, thx, it should be `{{CredentialCreationOptions}} or {{CredentialRequestOptions}}` --- because "relevant credential interfaces" might be called with either.

done in eed4835